Classes/MOC/Varnish/Package.php

```diff
@@ -2,18 +2,26 @@
 namespace MOC\Varnish;
 
 use TYPO3\Flow\Package\Package as BasePackage;
+use TYPO3\Flow\Annotations as Flow;
 
 class Package extends BasePackage {
 
 	/**
+     * @Flow\InjectConfiguration("enabled")
+     * @var boolean
+     */
+    protected $enabled;
```

Comment:
yes, had also problems at that point, so i changed to:
public function injectSettings(array $settings)

but that run into an error so i decided to put that into the target function:

MOC\Varnish\Service\ContentCacheFlusherService', 'flushForNode'

I will figure it out whats wrong there...